.travis.yml

```diff
@@ -1,22 +1,41 @@
-language: python
-python:
-  - "3.6"
-  - "3.7"
-  - "3.8"
-addons:
-  apt:
-    update: true
-env:
-  - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
-before_install:
-  - sudo add-apt-repository -y ppa:duggan/bats
-  - sudo apt-get -y install bats
-install:
-  - pip install pip -U && pip install -r requirements-dev.txt -U
-script:
-  - bats test-package.bats
-  - ./analyse-source-code.sh
-after_success:
-  - coveralls
-notifications:
-  email: false
+build:
```

Comment:
This section should build a package only, I would suggest leaving only the following config here 
```
  language: python
  python:
    - "3.6"
    - "3.7"
    - "3.8"
  addons:
    apt:
      update: true
  env:
    - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
install:
 - pip install pip -U
script:
  - python setup.py install
```

Reply:
Is it appropriate to make separate stages with `jobs: include` keywords? something like:
```
jobs:
  include:
    - name: "Build package"
    - name: "Tests running"
```

Reply:
Sure, please take a look here https://docs.travis-ci.com/user/build-stages/define-steps/

Reply:
stage unittests was added